spacepy/irbempy/irbempy.py

```diff
@@ -1455,13 +1470,13 @@ def _get_Lstar(ticks, loci, alpha, extMag='T01STORM', options=[1,0,0,0,0], omniv
     mlt[np.where( tb.feq(mlt,d['badval'])) ] = np.NaN
 
     results = {}
-    if nalpha == 0:
-        results['Lm'] = lm[0:nTAI]
-        results['Lstar'] = lstar[0:nTAI]
+    if no_shell_splitting:
+        results['Lm'] = lm[0:nTAI][:,None]
```

Comment:
Why is the extra axis required on here now when it wasn't before? Is it because this code was never reachable before (with the `raise Exception`)?

Reply:
Yes, previously the code only called the *_shell_splitting version, which I believe returns arrays with different number of dimensions. The returned arrays are post-processed in the `reassemble` function, but we need to ensure that the routines have the same number of dimensions for both versions.